test/compoundManager_invest.js

```diff
@@ -25,227 +25,227 @@ const ETH_TOKEN = '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE';
-            }
-            else {
-                await cEther.from(borrower).mint({ value: parseEther('2') });
-                tx = await cToken.from(borrower).borrow(parseEther('0.1'));
-                txReceipt = await cToken.verboseWaitForTransaction(tx);
-                assert.isTrue(await utils.hasEvent(txReceipt, cToken, "Borrow"), "should have generated Borrow event");
-            }
-            // increase time to accumulate interests
-            await manager.increaseTime(3600 * 24 * days);
-            await cToken.accrueInterest();
-            await cEther.accrueInterest();
-        }
-
-        describe("Add Investment", () => {
-
-            it('should invest in ERC20 for 1 year and gain interests (blockchain tx)', async () => {
-                await addInvestment(token.contractAddress, parseEther('1'), 365, false);
-            });
-
-            it('should invest in ERC20 for 1 year and gain interests (relay tx)', async () => {
-                await addInvestment(token.contractAddress, parseEther('1'), 365, true);
-            });
-
-            it('should invest in ETH for 1 year and gain interests (blockchain tx)', async () => {
-                await addInvestment(ETH_TOKEN, parseEther('1'), 365, false);
-            });
-
-            it('should invest in ETH for 1 year and gain interests (relay tx)', async () => {
-                await addInvestment(ETH_TOKEN, parseEther('1'), 365, true);
-            });
+      function testRemoveERC20Investment(fraction, relay) {
+        it(`should remove ${fraction / 100}% of an ERC20 investment (${relay ? "relay" : "blockchain"} tx)`, async () => {
+          await removeInvestment(token.contractAddress, fraction, relay);
         });
-
-        describe("Remove Investment", () => {
-
-            function testRemoveERC20Investment(fraction, relay) {
-                it(`should remove ${fraction / 100}% of an ERC20 investment (${relay ? "relay" : "blockchain"} tx)`, async () => {
-                    await removeInvestment(token.contractAddress, fraction, relay);
-                });
-            }
-            function testRemoveETHInvestment(fraction, relay) {
-                it(`should remove ${fraction / 100}% of an ETH investment (${relay ? "relay" : "blockchain"} tx)`, async () => {
-                    await removeInvestment(token.contractAddress, fraction, relay);
-                });
-            }
-
-            for (i = 1; i < 6; i++) {
-                testRemoveERC20Investment(i * 2000, true);
-                testRemoveERC20Investment(i * 2000, false);
-                testRemoveETHInvestment(i * 2000, true);
-                testRemoveETHInvestment(i * 2000, false);
-            }
+      }
+      function testRemoveETHInvestment(fraction, relay) {
+        it(`should remove ${fraction / 100}% of an ETH investment (${relay ? "relay" : "blockchain"} tx)`, async () => {
+          await removeInvestment(ETH_TOKEN, fraction, relay);
```

Comment:
Not sure why the diff is so large for this file, but this is the only line I changed: passing `ETH_TOKEN` (instead of `token.contractAddress`) to `removeInvestment()`